components/automate-chef-io/content/docs/eas_applications.md

```diff
@@ -0,0 +1,133 @@
+In the Chef Automate UI, create an API token as described in the Chef Automate documentation. Save this token somewhere safe.
+Please see (https://www.habitat.sh/docs/habitat-cli/#hab-sup-run "habitat documentation") for more habitat options.
+
+On the systems that runs Chef Habitat, you will need to launch the habitat supervisor with the following options:
+
+```
+HAB_FEAT_EVENT_STREAM=1 hab sup run \
+--event-stream-application=MY_APP \
+--event-stream-environment=MY_ENV \
+--event-stream-site=MY_SITE \
+--event-stream-url=AUTOMATE_HOSTNAME:4222 \
+--event-stream-token=API_TOKEN
+```
+
+Replace MY_APP, MY_ENV, MY_SITE, AUTOMATE_HOSTNAME, and API_TOKEN with the appropriate values. MY_APP, MY_ENV, MY_SITE are user-defined and can be set to any string. MY_APP, MY_ENV are used to group services in the Chef Automate interface and are intended to be used to describe your applications and application lifecycle, as explained in the concepts section of this document. MY_SITE is intended to describe the physical (i.e., datacenter) or cloud-specific (i.e., AWS region) location where your services are deployed. These values are search enabled, allowing for filtering of services.
+
+## Lifecycle management
+A future release will enable you to manage the lifecycle of disconnected services via the UI. Currently the functionality is available via the command line and using the API directly. Using the command line will kick off a one time job to assess disconnection status or delete disconnected services. The API is used to manage the activity of periodic jobs to assess disconnection status and deletion.
+
+Disconnected services are defined as services known to the EAS Applications backend storage that have not received a health check message for a user-defined period of time.
+
+### Periodic disconnection checks
+
+Automate assesses the status of the habitat services based on the receipt of a health check message. If a health check message has not been recieved for a configurable amount of time the service is marked as disconnected. The default amount of time is 5 minutes and this check cannot be disabled. Habitat send the health check messages every 30 seconds by default, this is also configurable using the `--health-check-interval` option with habitat supervisor. Please see (https://www.habitat.sh/docs/habitat-cli/#hab-sup-run "habitat documentation") for more information
+
+To configure the threshold for marking a disconnected service, you can do this via the UI or the API directly. To access the configuration settings in the UI, go to the settings tab, then Data Lifecycle on the sidebar. Under service groups you will find settings for Marking and Deleting disconnected services.
+
+Below is an example of using the API directly:
+
+```bash
+curl -sSX POST "https://automate-url/apis/v0/retention/service_groups/disconnected_services/config" -d
+'{
+  "threshold": "15m"
+}'
+-H "api-token: $TOKEN"
+```
+
+### Periodic deletion
```

Comment:
Minor: replacing _deletion_ with _removal_ to match with the language used in the settings UI